### PR TITLE
fix: [PL-27815]: Add Error Framework to LDAP wizard

### PR DESCRIPTION
**Pull request by upadhyay-an**
### Summary

1. Use ErrorHandler component to show errors on LDAP Wizard
2. Change button styles on Group & User edit

#### Screenshots

connectionSettingsErrorHandler

query-edit-btn

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

## [Contributor license agreement](https://github.com/harness/harness-core-ui/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 17 Issues 
* Code issue 17 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10995)

**Comment by upadhyay-an** [comment]
retrigger titlecheck

File: src/modules/33-auth-settings/components/CreateUpdateLdapWizard/views/StepGroupQueries.tsx [hunk 1]
```diff
@@ -102,11 +108,29 @@ const GroupQueryEdit: React.FC<
       if (result.resource?.status === 'SUCCESS') {
         setGroupQueryTestResult(<QueryTestSuccessMsg message={getString('authSettings.ldap.queryTestSuccessful')} />)
       } else {
-        setGroupQueryTestResult(<QueryTestFailMsg message={result.resource?.message} />)
+        setGroupQueryTestResult(
+          <QueryTestFailMsgs
+            errorMessages={
+              (result?.responseMessages && result?.responseMessages?.length > 0 && result?.responseMessages) ||
```
Comment (pbarapatre10) [inline src/modules/33-auth-settings/components/CreateUpdateLdapWizard/views/StepGroupQueries.tsx:114]: minor This condition seems redundant, thanks

**Review by pbarapatre10 — APPROVED** [review]

File: src/modules/33-auth-settings/components/CreateUpdateLdapWizard/views/StepGroupQueries.tsx (hunk 1 above)
Comment (upadhyay-an) [reply src/modules/33-auth-settings/components/CreateUpdateLdapWizard/views/StepGroupQueries.tsx:114]: The code doesn't give a clear idea but it's trying to do is check if resposeMessages aren't undefined or null, then checks if array is not empty and only then returns the response messages by short circuiting.  (this explains all three checks, let me know if this makes sense or there is a better way to put this.

Comment (pbarapatre10) [reply src/modules/33-auth-settings/components/CreateUpdateLdapWizard/views/StepGroupQueries.tsx:114]: Understood! thanks for explaining

**Review by reetika04 — APPROVED** [review]

**Pull request merged**